packaging/language/composer.py

```diff
@@ -116,6 +132,8 @@ def parse_out(string):
     return re.sub("\s+", " ", string).strip()
 
 def has_changed(string):
+    if string == "":
```

Comment:
hmm I see. Do you have an example?

Generally we should return `changed=True` if we are not sure that something has changed, also for commands executed `changed=True` makes sense.